packages/xchain-thorchain/src/thor/types.ts

```diff
@@ -0,0 +1,59 @@
+import { BigSource } from 'big.js'
+
+import { Asset, THORChain } from '@xchainjs/xchain-util'
+import { PrivKey, Msg } from 'cosmos-client'
+import { BaseAccount, StdTx } from 'cosmos-client/x/auth'
+
+export const AssetThor: Asset = { chain: THORChain, symbol: 'THOR', ticker: 'THOR' }
```

Comment:
Don't we have RUNE here ^ (similar to [AssetRuneNative](https://gitlab.com/thorchain/asgardex-common/asgardex-util/-/blob/master/src/asset.ts#L120)
```
const AssetRune: Asset = { chain: THORChain, symbol: 'RUNE', ticker: 'RUNE' }
```
```